### PR TITLE
Use LAN ip address

### PR DESCRIPTION
**Pull request by nelsonic**
Now, instead of requiring people to load the **client.js** script from CDN they have the option to load it from a **Local** web server!
See: https://github.com/ideaq/faster/issues/34 (encourage people to use **LAN IP ADDRESS**)

faster-boot-console-explain-basedir

Which means that if the internet is patchy or if people want to use **_Faster**_ **100% OFFLINE** they can! :rocket:

### CODE REVIEW

**Comment by nelsonic** [comment]
@iteles Quite a lot has changed since _yesterday_, let me know if you have questions. :+1:

File: example/index.html [hunk 1]
```diff
@@ -1,12 +1,13 @@
 <html>
   <head>
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
-    <title>Faster Re-Load</title>
+    <title>Hello Ines</title>
```
Comment (iteles) [inline example/index.html:4]: Immortalised forever in code :wink: :tada: 
You could probably make this whole sample index a bit more generic/explanatory though - leaving in the kittens _of course_ .

Comment (nelsonic) [inline example/index.html:4]: Indeed! We could totes have a much nicer **example/index.html**
Want to contribute one? :-)

**Comment by nelsonic** [comment]
@iteles  any other questions? :wink:

File: example/index.html (hunk 1 above)
Comment (iteles) [inline example/index.html:4]: Yes!

**Comment by iteles** [comment]
Nope, this is **awesome** :tada:

**Pull request merged**